test/TensorFlowRuntime/accelerator_only.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %target-run-dynamic-compilation-swift
+// RUN: %target-swift-frontend -emit-sil -O %s -verify | %FileCheck %s
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+
+import TensorFlow
+import TensorFlowUnittest
+import StdlibUnittest
+
+@TensorFlowGraph
+func add_constant(_ a : Tensor<Float>) -> Tensor<Float> {
+  return a + Tensor<Float>([[5]])
+}
+
+// CHECK-LABEL: sil hidden [noinline] @{{.*}}accelerator_only14matmul_and_add{{.*}}
```

Comment:
```suggestion
// CHECK-LABEL: @{{.*}}accelerator_only14matmul_and_add{{.*}}
```

We usually drop the front of the declaration and just leave the function name for two reasons:
1. The compiler may emit different attributes on SIL declarations on different platforms (I've encountered that on macOS).
2. When we merge from upstream, attributes may change. Leaving just the function name would avoid that.